core/Command/Security/BruteforceAttempts.php

```diff
@@ -41,11 +41,11 @@ protected function configure(): void {
 		parent::configure();
 		$this
 			->setName('security:bruteforce:attempts')
-			->setDescription('resets bruteforce attempts for given IP address')
+			->setDescription('list bruteforce attempts for given IP address')
```

Comment:
```suggestion
			->setDescription('List bruteforce attempts for given IP address')
```

But it's not actually a list. More like the statistics. So maybe something like:

```suggestion
			->setDescription('Bruteforce attempts information for given IP address')
```

Reply:
Agreed. "list" came from the existing Admin Manual. (I'll update the final choice over there when this is done).
```suggestion
			->setDescription('show bruteforce attempts status for a given IP address')
```